app/brave_generated_resources.grd

```diff
@@ -361,6 +364,11 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         Block elements
       </message>
 
+      <!-- Adblock contenxt menu -->
+      <message name="IDS_NEW_EMAIL_ALIAS" desc="Message for context menu that enables the Email Alias generator UI for the page">
+        New Email Alias
```

Comment:
Do we want a title case / non title case version of this if its showing up in the context menu?

Reply:
Yes, renderer context menu also needs 2 versions.